### PR TITLE
Add a demo for injecting error to a container

### PR DESCRIPTION
**Pull request by yuanchen8911**
#### What type of PR is this?

/kind feature
#### What this PR does / why we need it:
This PR adds a basic demo for injecting container error using kwokctl. I'll file another PR for a more realistic in-cluster demo with virtual GPU nodes.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: demo/container-failure.svg
Comment (wzshiming) [inline demo/container-failure.svg:1]: Use `rm demo/container-failure.svg; make demo/container-failure.svg` made this

Comment (yuanchen8911) [reply demo/container-failure.svg:1]: Updated it.

**Comment by yuanchen8911** [comment]
Updated and recreated the svg file.

**Review by wzshiming — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**